packages/wallet-sdk/src/core/communicator/util.ts

```diff
@@ -14,10 +14,15 @@ export function openPopup(url: URL): Window {
     'Smart Wallet',
     `width=${POPUP_WIDTH}, height=${POPUP_HEIGHT}, left=${left}, top=${top}`
   );
-  popup?.focus();
-  if (!popup) {
+
+  if (popup) {
+    popup.focus();
+  } else {
+    // The popup may fail to open if the user has a pop-up blocker enabled
+    alert('Please disable your pop-up blocker and try again');
```

Comment:
it's not a scope of this pr, but would it be possible to listen to user event on alert closing and retry popup open? @arjun-dureja

Reply:
I think it's technically possible, but may cause problems because the user would need to close the alert before they can go to their settings to disable popup blocking. So if we try again immediately after closing, it'll just result in another alert/failure

Reply:
i mean, i think we can make the popup open call directly and immediately to get triggered via a specific user action, so theoretically browsers should not block it, no?

Reply:
web noob question.
can we use `confirm()` and retry opening when it returns true?